Enigmatry.Entry.AspNetCore.Tests.NewtonsoftJson.Tests/Enigmatry.Entry.AspNetCore.Tests.NewtonsoftJson.Tests.csproj

```diff
@@ -1,10 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net8.0</TargetFramework>
-    <ImplicitUsings>enable</ImplicitUsings>
-    <IsPackable>false</IsPackable>
-    <Nullable>enable</Nullable>
+    <IsTestProject>true</IsTestProject>
```

Comment:
According to the documenation IsTestProject helps all tools to figure out that this is test project (I have added this to all missing projects)